### DISCUSSION TITLE
Simple lists are unable to be styled?

### DISCUSSION THREAD

**Discussion by kevinmonisit** [tailwindlabs/tailwindcss#9057] [Help] [answered]
When attempting to style an `<ol>` with `class="list-decimal"`, there is no style. This can be re-produced on https://play.tailwindcss.com/. Am I doing something wrong here? This is on v3.1.5 of Tailwindcss

![Screen Shot 2022-08-09 at 11 11 42 AM](https://user-images.githubusercontent.com/7038712/183687804-7c09f682-5618-4118-bf95-1c1e73caf7d7.png)

**Comment by wongjn** [comment, marked as answer]
In Tailwind's preflight (reset) styles, list styles are reset like so:

```css
ol, ul, menu {
  list-style: none;
  /* … */
}
```

`list-style` is shorthand, and this means that implicitly, [`list-style-position` is set to its initial value of `outside`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position#formal_definition). If we look back at your example it seems like the `list-decimal` has had no effect, but actually, it is because the markers are positioned `outside`. [Here's a Tailwind Play](https://play.tailwindcss.com/PoxENel9oH) showcasing this is the case; the first list has some left margin so we can see the `outside`-positioned markers, and the second list has `list-inside` to bring the marker positions `inside`.

**Reply by kevinmonisit** [reply]
Thank you for your in-depth answer!
